src/main/java/org/prebid/server/spring/config/PrivacyServiceConfiguration.java

```diff
@@ -53,6 +53,7 @@
     VendorListServiceV1 vendorListServiceV1(
             @Value("${gdpr.vendorlist.v1.cache-dir}") String cacheDir,
             @Value("${gdpr.vendorlist.v1.http-endpoint-template}") String endpointTemplate,
+            @Value("${gdpr.vendorlist.v1.deprecated:#{false}}") boolean deprecated,
```

Comment:
1. Place after `fallbackVendorListPath` argument.
2. Update `/docs/config-app.md` with this new configuration option.
3. According to project code style we prefer to avoid defaults in `@Value`s. Let it be defined in application.yaml explicitly.